webapp/src/main/resources/public/js/components/workbench/TextUnit.js

```diff
@@ -841,17 +868,66 @@ let TextUnit = createReactClass({
         );
     },
 
+    renderRepository() {
+        if (this.props.viewMode.viewMode !== ViewModeStore.VIEW_MODE.FULL) {
+            return;
+        }
+
+        return (
+            <Label bsStyle='danger' bsSize='large' className="clickable" onClick={this.onRepositoryClick}>
```

Comment:
I don't think we should be using the "danger" style here since it is just regular information. see the comment on the whole PR. Maybe having a prefix on the label to say "repository" or "repo" and put it on its own line?

Reply:
Also, adding a new line is not that easy, since most of the space is already used. So adding a new line in the label, would make each text line even bigger.

Would you be fine with using the `secondary` color for the repository?

Reply:
btw, I just remembered that the asset info is already accessible by hovering on the text unit (and clicking on the icon drills down to search in that asset). I guess that's the approach i had to avoid cluttering everything at the time. 

we already have a few label, eg. "unused" can be added to the right of the locale.

I think we should keep the danger color for things that are actually problematic and not for regular display. 

In the full view, we can use more line for clarify, I think. Maybe something like. 

<img width="272" alt="Screenshot 2024-02-28 at 2 20 00 PM" src="https://github.com/box/mojito/assets/9702182/ee95e736-eba9-4e87-9fca-61025148c285">

Reply:
Full:

![tmp](https://github.com/box/mojito/assets/3407462/91da2f93-b1f5-4e29-bb43-d8bf61ab602a)

Reduced:

![tmp](https://github.com/box/mojito/assets/3407462/252b4aa7-e7fa-4e05-9d6e-42e198d8af48)